### DISCUSSION TITLE
Crackling sound related to sample rate

### DISCUSSION THREAD

**Discussion by joaquinmir** [pschatzmann/arduino-audio-tools#947] [Q&A] [answered]
Hi! I'm testing this library and i have problems running the "streams-generator-i2s.ino" example file. I hear the sine wave but it sounds glitchy, the sound gets cleaner when i set the sample rate at around 35 kHz in the AudioInfo constructor . The crackling increases with the sample rate.
I'm using the ESP32 DEV KIT V1 with the PCM5102 External DAC.
This is how the wave look like with sample rate at 41KHz (default value):
![Captura de pantalla 2023-07-28 042500](https://github.com/pschatzmann/arduino-audio-tools/assets/97806069/3a426b76-9f8f-4fc8-8eef-643bceb86ed5)

**Comment by pschatzmann** [comment, marked as answer]
Can it be that you just ignored all recommendations in regard of setting the log level?
I never had any issues even with 48000 samples per secong
